Source/Session/Session.swift

```diff
@@ -62,6 +62,9 @@ public class Session: NSObject {
         }
         
         let visit = makeVisit(for: visitable, options: options ?? VisitOptions())
+        if self.webView.uiDelegate == nil && visit.visitable.visitableViewController is WKUIDelegate {
```

Comment:
The only `UIViewController` in the project is `VisitableViewController`, and I need a `UIViewController` so I can call `present` to display the `UIAlertController`.

A new `VisitableViewController` typically gets created for every `Session.visit` call. Hence I need to re-assign the delegate on every visit, which is why I have this funky looking code here.

A user overriding the delegate will have a much better time, because they can just set the delegate once when the session is created, assuming they have a long lasting view controller (eg. a `UINavigationController`). But there's no guarantees within the framework that such a controller will exist.

I'm by no means a Swift/iOS expert. Let me know if any of this is totally wrong :)

Reply:
Oh, and the `self.webView.uiDelegate == nil` check means that if a user has set the delegate *before* calling `Session.visit` (eg. on Session init, as I suggest in docs), then this will do nothing.